src/concepts/config-intro.md

```diff
@@ -82,9 +107,13 @@ In the [http-kafka binding](../reference/config/bindings/binding-http-kafka.md),
 
 A corresponding `routes[].when` object with a matching `GET` method and `location` path is also required for follow-up `GET` requests to return the same response as would have been returned if the `prefer: respond-async` request header had been omitted.
 
+### Route Exit
+
+A route exists to direct messages on the stream to a desired exit point. This is the next binding needed to parse the the stream data. Bindings like [tcp](../reference/config/bindings/binding-tcp.md) are frequently used to route incoming streams to different exit points.
+
 ### Guarded Routes
 
-A route is considered guarded if a [guard](#guards) is specified. Any guard can be configured, enabling different use cases when protecting data sent over a stream.
+A route is considered guarded if a [guard](#guards) is specified. Any guard can be configured, enabling different use cases when protecting data sent over a stream. The guard condition supersedes any other conditions and is evaluated first.
```

Comment:
Supersedes may not be the best word here. Logically the guard is anded with the when conditions, so order of evaluation doesn't matter so much, guarded false ands to give false if evaluated first or last. The main point is that guarded routes cannot be followed if the guarded check fails. This allows for fall through to a different route based on authorized roles and privileges.